series-advisor/src/index.css

```diff
@@ -0,0 +1,73 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-image: url(https://assets.nflxext.com/ffe/siteui/vlv3/02dd3f21-2c34-4989-a48b-827dd9edc446/6fd9c935-5097-4045-9e29-4e1028a6c1fb/PL-pl-20190422-popsignuptwoweeks-perspective_alpha_website_large.jpg);
+
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
+
+.login-container {
+  background-color: rgba(0, 0, 0, 0.85);
+  max-width: 25%;
+  margin: 10% auto;
+}
+
+.login-container__form {
+  display: flex;
+  flex-direction: column;
+  padding: 2rem 5rem
+}
+
```

Comment:
Except of `login-container__others`, every class name below this line is not valid in terms of BEM. Compare them with previous projects and correct.